frontend/src/modules/history/components/Translation.js

```diff
@@ -251,172 +240,202 @@ export class TranslationBase extends React.Component<InternalProps, State> {
+                                    title={ `${translation.date} UTC` }
+                                />
+                            </div>
+                            <menu className='toolbar'>
 
-                        { this.renderDiffToggle() }
+                            { this.renderDiffToggle() }
 
-                        { (!translation.rejected || !canDelete ) ? null :
-                            // Delete Button
                             <Localized
-                                id='history-Translation--button-delete'
+                                id='history-Translation--button-comment'
                                 attrs={{ title: true }}
                             >
                                 <button
-                                    className='delete far'
-                                    title='Delete'
-                                    onClick={ this.delete }
-                                    disabled={ this.props.isActionDisabled }
-                                />
-                            </Localized>
-                        }
-                        { translation.approved ?
-                            // Unapprove Button
-                            ( canReview && !isReadOnlyEditor ) ?
-                                <Localized
-                                    id='history-Translation--button-unapprove'
-                                    attrs={{ title: true }}
+                                    className={ commentCount === 0 ? 'comment-btn' :
+                                        'comment-btn comment-count'
+                                    }
+                                    title='Comment'
+                                    onClick={ this.toggleComments }
                                 >
-                                    <button
-                                        className='unapprove fa'
-                                        title='Unapprove'
-                                        name='unapprove'
-                                        onClick={ this.handleStatusChange }
-                                        disabled={ this.props.isActionDisabled }
-                                    />
-                                </Localized>
-                                :
-                                <Localized
-                                    id='history-Translation--button-approved'
-                                    attrs={{ title: true }}
-                                >
-                                    <button
-                                        className='unapprove fa'
-                                        title='Approved'
-                                        disabled
-                                    />
-                                </Localized>
+                                    { commentCount === 0 ?'Comment' : `${commentCount} Comment` }
```

Comment:
This text will need to be localized. For inspiration, see https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/batchactions/components/ApproveAll.js#L57. In addition to the `one` and `other` forms, you should also add `0` with a value `Comment`.